resources/views/registerProject.blade.php

```diff
@@ -0,0 +1,218 @@
+@extends('layouts.app')
+
+@section('title', 'Add Project')
+
+@section('content')
+<!-- Internal CSS stylesheet -->
+<style>
```

Comment:
Please move your styles to resources/sass/app.scss

Reply:
Even if its very minimal styling and specific for this view only? From what I understood, if it's minimal then it's better if we leave it as internal styling because it takes less effort to get the styling from externally.

Reply:
Si es algo que probablemente se usaría en otro forms accross la plataforma, es preferible que esté en el .sass